integration/test-cleanup.sh

```diff
@@ -11,9 +11,27 @@ for app in $( echo "${apps}" | jq -r '.[] | @base64' ); do
     name=$(_jq '.Name')
 		# TODO(jfatta): should remove only those 
 		# created during the same test session
-    if [[ $name = integration-test-* ]]
+    if [[ $name = integration-test-app-* ]]
     then
         echo deleting "$name"
         $( auth0 apps delete "$clientid")
     fi
 done
+
+apis=$( auth0 apis list --format json --no-input )
+
+for api in $( echo "${apis}" | jq -r '.[] | @base64' ); do
+    _jq() {
+     echo "${api}" | base64 --decode | jq -r "${1}"
+    }
+
+    id=$(_jq '.ID')
+    name=$(_jq '.Name')
+		# TODO(jfatta): should remove only those 
+		# created during the same test session
+    if [[ $name = integration-test-api-* ]]
+    then
+        echo deleting "$name"
+        $( auth0 apis delete "$id")
+    fi
+done
```

Comment:
This can be parallelized in batches of 5 with
```sh
auth0 apis list --format json --no-input \
    | jq -r '.[] | select(.Name | startswith("integration-test-api-")) | "\(.Name) \(.ID)\n"' \
    | xargs -P 5 -n 2 bash -c 'echo Deleting $0 && auth0 apis delete $1 --force' 
```